timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -279,7 +277,7 @@ void declare_context(py::module &m) {
             std::vector<unsigned long long> du_dx(N * D);
             ctxt.get_du_dx_t_minus_1(&du_dx[0]);
             py::array_t<double, py::array::c_style> py_du_dx({N, D});
-            for (int i = 0; i < du_dx.size(); i++) {
+            for (unsigned int i = 0; i < du_dx.size(); i++) {
```

Comment:
I guess we could also use `auto` here?

Reply:
Can also remove this function entirely soon, based on this comment: https://github.com/proteneer/timemachine/blob/06299a7f60106eb1b04de625408ec9f30929b960/timemachine/cpp/src/wrap_kernels.cpp#L273 (but would need to edit also [test_md.py](https://github.com/proteneer/timemachine/blob/06299a7f60106eb1b04de625408ec9f30929b960/tests/test_md.py#L176-L179))

Reply:
`auto` is fine here as well.

Reply:
`size_t` is also fine.